week-1/implement/sum.js

```diff
@@ -0,0 +1,25 @@
+function sum(arr) {
+  if (arr.length === 0) {
+    return 0;
+  }
+
+  let totalSum = 0;
+
+  for (let i = 0; i < arr.length; i++) {
+    const num = arr[i];
+    if (!isNaN(num)) {
+      totalSum += num;
+    }
+  }
+
+  return totalSum;
+}
```

Comment:
Theres is another method of doing this by creating another function inside this function. Can you figure out how to do that? If not, I can help show you what I did.